solr/core/src/java/org/apache/solr/handler/StreamHandler.java

```diff
@@ -141,15 +141,21 @@ public static void addExpressiblePlugins(StreamFactory streamFactory, SolrCore c
   }
 
   public static class ExpressibleHolder extends PackagePluginHolder<Class<? extends Expressible>> {
+    private Class<? extends Expressible> clazz;
+
     public ExpressibleHolder(PluginInfo info, SolrCore core, SolrConfig.SolrPluginInfo pluginMeta) {
       super(info, core, pluginMeta);
     }
 
-    // WARNING: This is called from the super constructor, do not make assumptions about object state.
-    // This will probably cause a really bad bug someday, I hope you are reading this under better circumstances.
+    @Override
```

Comment:
The commit "potential class loading issue" is not related to the PR

Reply:
It is, because it was a bug I introduced in this issue.